ironfish/src/primitives/transaction.ts

```diff
@@ -112,9 +116,8 @@ export class Transaction {
 
     // sender address
     // TODO(joe): read from bytes rather than hardcoded value
-    // this._senderAddress = reader.readBytes(PUBLIC_ADDRESS_LENGTH, true).toString('hex')
+    // this._sender = reader.readBytes(PUBLIC_ADDRESS_LENGTH, true).toString('hex')
```

Comment:
What is the plan to replace the hardcode value?  Anything blocks this change? @jowparks